packages/esm-patient-registration-app/src/patient-registration/input/custom-input/identifier/identifier-input.component.tsx

```diff
@@ -14,6 +14,7 @@ interface IdentifierInputProps {
 }
 
 export const IdentifierInput: React.FC<IdentifierInputProps> = ({ identifierType }) => {
+  const { setFieldValue } = useContext(PatientRegistrationContext);
```

Comment:
Can you merge this line with line 28 (`const { validationSchema, setValidationSchema } = React.useContext(PatientRegistrationContext);`)? There's no reason to use the same context twice. Ideally line 28 is removed (you can just add `validationSchema` and `setValidationSchema` to line 17.
Thanks!